bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/TelescopePosTableWidget.java

```diff
@@ -918,14 +918,20 @@ void updatePrimaryStar() {
         if (_env == null || !OTOptions.isEditable(_obsComp.getProgram(), _obsComp.getContextObservation())) return;
 
         // TODO: This method must be tested in greater detail, since custom guiding is required for offset positions.
```

Comment:
If nobody remembers it or understand what's it about we may as well remove it, it may lead to confusions